pyvista/utilities/geometric_objects.py

```diff
@@ -688,3 +689,58 @@ def CircularArc(pointa, pointb, center, resolution=100, normal=None,
 
     arc.Update()
     return pyvista.wrap(arc.GetOutput())
+
+
+def Pyramid(points):
+    """Create a pyramid defined by 5 points.
+
+    The pyramid has 5 points. The user can specify each points.
+
+    Parameters
+    ----------
+    points : np.ndarray or list
+        List of Position of the points.
```

Comment:
```suggestion
        Points of the pyramid.  Points are ordered such that the first
        four points are the four counterclockwise points on the
        quadrilateral face, and the last point is the apex.
```
Best to describe the order of the points.